examples/bukkit/commandtrees/src/main/java/io/github/jorelali/BetterBossBarsCommand.java

```diff
@@ -66,7 +66,7 @@ public void registerBetterBossBarCommand() {
 						)
 					)
 					.then(new LiteralArgument("style")
-						.then(new MultiLiteralArgument("styles", List.of("notched_6", "notched_10", "notched_12", "notched_20", "progress"))
+						.then(new MultiLiteralArgument("styles", "notched_6", "notched_10", "notched_12", "notched_20", "progress")
```

Comment:
After seeing this change, I'm no longer happy with this example plugin anymore. Given the changes that we've made to support named nodes, we're still accessing arguments by index in our executors:

```java
NamespacedKey id = (NamespacedKey) args.get(0);
String style = (String) args.get(1);
```

Would it be unreasonable to try and push for accessing arguments by their node names as the default convention?